doc/Setup/Getting_started_on_your_laptop.rst

```diff
@@ -0,0 +1,179 @@
+##################################
+Getting Started on an Apple Laptop
+##################################
+
+This page contains our notes on how to set your Mac laptop up to run the LSST DM stack and sims tools. 
+
+.. contents::
+   :depth: 4
+
+
+Installing the DM Stack and Sims Tools
+======================================
+The Twinkles package uses the LSST DM Stack's `afw` code, and also the LSST
+Sims tools (among other things). Here's how to get everything working.
+
+Get Anaconda Python
+--------------------------------
+
+Click `here <http://conda.pydata.org/miniconda.html>`_ and download python 
+2.7, if you don't have Anaconda python already.
+
+.. code-block:: bash
+
+    bash ~/Downloads/Miniconda-latest-MacOSX-x86_64.sh
+
+This installs miniconda into `${HOME}/miniconda2` by default. You should 
+now pre-pend your `PATH` with `${HOME}/miniconda2/bin` so that this becomes
+your default version of python.
+
+Install the LSST DM Stack and LSST Sims Tools
+---------------------------------------------
+These instructions come from the `LSST Science
+Pipelines <https://pipelines.lsst.io/install/conda.html>`_. The first thing that they recommend is to make sure `conda` is up to date:
+
+.. code-block:: bash
+
+   conda update conda
+
+The following commands will download and activate the current release versions of the LSST Science Pipelines in a 
+new Conda environment named "lsst". At various times they need to to type "`y`", so unfortunately you cannot leave them too long. They 
+each take a few minutes, except for the `conda install`, which takes about an hour.  
+
+.. code-block:: bash
+
+   conda config --add channels http://conda.lsst.codes/stack  
+   conda create --name lsst python=2
+   
+You are now ready to install the `lsst` software. 
+
+If you are a C-shell user, the `source activate` command below
+will not work, as that script only supports `bash` and `zsh`. A workaround is to switch to a `bash` shell at this point (by typing 
+`bash`), and then stay in that `bash` shell to do the other two commands below. It's not really a workaround though - because you 
+will have to come back to `bash` and re-do this environment set-up every time you want to use the LSST software anyway.
+
+.. code-block:: bash
+
+   source activate lsst
+   conda install lsst-distrib lsst-sims
+   source eups-setups.sh
+
+You should now be set up to use the DM Stack in the current `bash` shell.
+
+Set Up Environment to Use LSST Code
+-----------------------------------
+Every time you start a new shell, you need to set up its environment to enable use of the LSST code. The following lines could be 
+worth adding to your `~/.bash_profile` file or equivalent.
```

Comment:
Historically, adding LSST setup commands to ~/.bash_profile has caused problems for people.  Setting up the LSST stack usually involves using a specific installation of python, which is not always the user's default preferred python installation.  I would instead recommend that they copy those commands to an ```setUpLSST.sh``` script, and just ```source``` that script whenever they want to run the LSST stack.